### ISSUE TITLE
Console Route Print Fails When Table Includes Both IPv4 and IPv6 Routes

### ISSUE THREAD

**Issue by sn0wfa11** [rapid7/metasploit-framework#7912]
## Steps to reproduce
1. Start a Meterpreter session on a machine that has both IPv4 and IPv6 routes available.
2. Add an IPv4 route for that session in the main console.
3. Add an IPv6 route for that session in the main console.
4. Type `route`, hit enter.

(Output below.)

## Expected behavior
Should print both IPv4 and IPv6 Routes.

## Current behavior
Throws an error when the table includes both IPv4 and IPv6 routes. See Below.

### Reason for current behavior
The !sort function rex-text-0.2.11/lib/rex/text/table.rb cannot handle a table when it includes both IPv4 and IPv6 addresses.

## Fix
Alter the cmd_route function in metasploit-framework/lib/msf/ui/console/command_dispatcher/core.rb to use two tables. One for IPv4 and one for IPv6. Meterpreter's route command currently does this, probably for the same reason.

**Have a working fix ready to go. The PR for the fix will be submitted shortly.**

### Metasploit version
```
msf > version
Framework: 4.13.20-dev-9a5d5eec2e
Console  : 4.13.20-dev-9a5d5eec2e
```

### I installed Metasploit with:
- [ ] Kali package via apt
- [ ] Omnibus installer (nightly)
- [ ] Commercial/Community installer (from http://www.rapid7.com/products/metasploit/download.jsp)
- [X] Source install (Ruby 2.3.3)

### OS
Kali Rolling

## Error Display
```
msf > route
[*] There are currently no routes defined.
msf > route add 10.5.5.0 255.255.255.0 1
[*] Route added
msf > route

Active Routing Table
====================

   Subnet             Netmask            Gateway
   ------             -------            -------
   10.5.5.0           255.255.255.0      Session 1

msf > route add fd15:4ba5:5a2b:1008:: ffff:ffff:ffff:ffff:: 1
[*] Route added
msf > route
[-] Error while running command route: comparison of Array with Array failed

Call stack:
/home/jhale/.rvm/gems/ruby-2.3.3@metasploit-framework/gems/rex-text-0.2.11/lib/rex/text/table.rb:195:in `sort!'
/home/jhale/.rvm/gems/ruby-2.3.3@metasploit-framework/gems/rex-text-0.2.11/lib/rex/text/table.rb:195:in `sort_rows'
/home/jhale/.rvm/gems/ruby-2.3.3@metasploit-framework/gems/rex-text-0.2.11/lib/rex/text/table.rb:110:in `to_s'
/home/jhale/git/metasploit-framework/lib/msf/ui/console/table.rb:48:in `to_s'
/home/jhale/git/metasploit-framework/lib/msf/ui/console/command_dispatcher/core.rb:923:in `cmd_route'
/home/jhale/git/metasploit-framework/lib/rex/ui/text/dispatcher_shell.rb:430:in `run_command'
/home/jhale/git/metasploit-framework/lib/rex/ui/text/dispatcher_shell.rb:392:in `block in run_single'
/home/jhale/git/metasploit-framework/lib/rex/ui/text/dispatcher_shell.rb:386:in `each'
/home/jhale/git/metasploit-framework/lib/rex/ui/text/dispatcher_shell.rb:386:in `run_single'
/home/jhale/git/metasploit-framework/lib/rex/ui/text/shell.rb:205:in `run'
/home/jhale/git/metasploit-framework/lib/metasploit/framework/command/console.rb:48:in `start'
/home/jhale/git/metasploit-framework/lib/metasploit/framework/command/base.rb:82:in `start'
./msfconsole:48:in `<main>'
msf > route remove 10.5.5.0 255.255.255.0 1
[*] Route removed
msf > route

Active Routing Table
====================

   Subnet                 Netmask                Gateway
   ------                 -------                -------
   fd15:4ba5:5a2b:1008::  ffff:ffff:ffff:ffff::  Session 1

msf > 
```

**Comment by dmohanty-r7** [comment]
Great PR, @sn0wfa11. Printing routing table with both ipv4/ipv6 works for me as well.

```
msf exploit(handler) > route
[*] There are currently no routes defined.
msf exploit(handler) > route add 10.5.5.0 255.255.255.0 1
[*] Route added
msf exploit(handler) > route

IPv4 Active Routing Table
=========================

   Subnet             Netmask            Gateway
   ------             -------            -------
   10.5.5.0           255.255.255.0      Session 1

[*] There are currently no IPv6 routes defined.
msf exploit(handler) > route add fd15:4ba5:5a2b:1008:: ffff:ffff:ffff:ffff:: 1
[*] Route added
msf exploit(handler) > route

IPv4 Active Routing Table
=========================

   Subnet             Netmask            Gateway
   ------             -------            -------
   10.5.5.0           255.255.255.0      Session 1


IPv6 Active Routing Table
=========================

   Subnet                 Netmask                Gateway
   ------                 -------                -------
   fd15:4ba5:5a2b:1008::  ffff:ffff:ffff:ffff::  Session 1

msf exploit(handler) > route remove 10.5.5.0 255.255.255.0 1
[*] Route removed
msf exploit(handler) > route

IPv6 Active Routing Table
=========================

   Subnet                 Netmask                Gateway
   ------                 -------                -------
   fd15:4ba5:5a2b:1008::  ffff:ffff:ffff:ffff::  Session 1

[*] There are currently no IPv4 routes defined.
msf exploit(handler) >
```

**Comment by dmohanty-r7** [comment]
Fixed with #7913
